frontend/.eslintrc.json

```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": "next/core-web-vitals",
+  "rules": {
+    "no-unused-vars": "error",
+    "require-await": "error"
```

Comment:
what is require-await?

Reply:
think its something to do with async functions having an await expression or else error